djgentelella/management/commands/loaddevstatic.py

```diff
@@ -155,6 +155,21 @@ def handle(self, *args, **options):
             'chartjs':[
                 'https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js',
                 'https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.css'
+            ],
+
+            'mjolnic-bootstrap-colorpicker': [
+                'https://colorlib.com/polygon/vendors/mjolnic-bootstrap-colorpicker/dist/css/bootstrap-colorpicker.min.css',
+                'https://colorlib.com/polygon/vendors/mjolnic-bootstrap-colorpicker/dist/js/bootstrap-colorpicker.min.js'
+            ],
+            "img/": [],
+            "img/bootstrap-colorpicker": [
```

Comment:
There is no congruency on mjolnic-bootstrap-colorpicker  and img/bootstrap-colorpicker  are they 2 diferent libs ?

Reply:
They are the same, but when JS and CSS try to figure where are the images they look for  "img/bootstrap-colorpicke" directory